File: Onoicrm.Domain/Entities/Patient.cs
namespace Onoicrm.Domain.Entities;

public class Patient : Entity
{
    public string WhatsAppNumber { get; set; } = "";
    public string? JobPosition { get; set; } = "";
    public string? Address { get; set; } = "";
    public string? Description { get; set; } = "";
    public Gender GenderId { get; set; } = Gender.NotDefined;
    public bool? MailingConsent { get; set; }
    public string FullName { get; set; }
    public long? UserProfileId { get; set; }
    public long? ClinicId { get; set; }

    public InformationSource? InformationSource { get; set; }
    public long? InformationSourceId { get; set; }
    public DateTime? BirthDate { get; set; }
    protected override int GetClassId() => ClassNames.Patient;
}

File: Onoicrm.Front/src/modules/user-profile/Patient.ts
import {ClinicEntity} from "~/shared/entities/ClinicEntity";
import {UserProfile} from "~/modules/user-profile/UserProfile";

export class Patient extends ClinicEntity {
    public lastName: string = "";
    public fullName: string = "";
    public userId!: string;
    public birthDate: string | null = null;
    public whatsAppNumber: string = "";
    public address: string = "";
    public description: string = "";
    public informationSourceId: number | null = null
    public jobPosition: string = "";
    public stateId: number = 2;

    constructor(instance?: Partial<UserProfile>) {
        super();
        Object.assign(this, instance);
    }
}